paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -453,21 +457,16 @@ internal class CustomerSheetViewModel @Inject constructor(
             else -> error("${viewState.value} is not supported")
         }
     }
-
-    private fun addPaymentMethod(
-        paymentMethodSpec: LpmRepository.SupportedPaymentMethod,
-        formViewData: FormViewModel.ViewData,
+    private fun createAndAttach(
```

Comment:
Looks like you're missing a newline.

But more so, it seems like this is really not the place for this level of logic, it seems like it should be pushed into a helper.

Reply:
Would you prefer if I do this in this PR or would a future PR suffice?

Reply:
Based on the comments from Till, it seems like we're also missing test coverage. I'd like to see it, but I won't block on it.